server/svix-server/src/v1/endpoints/endpoint/mod.rs

```diff
@@ -330,14 +340,11 @@ pub struct EndpointOut {
     #[serde(rename = "filterTypes")]
     pub event_types_ids: Option<EventTypeNameSet>,
     pub channels: Option<EventChannelSet>,
-
-    pub id: EndpointId,
```

Comment:
Annoying that this had to be removed - but in order for `ModelOut` to work, you need an `id` field. And it was easier if `EndpointOut impl ModelOut` instead of `EndpointOutNoMetadata`. (And since `EndpointNoMetadata` is embedded and flattened in `EndpointOut`, both structs can't have an `id` field)

Reply:
I don't fully understand this. Shouldn't this just be common-endpoint-out?

Also, why do we even need an EndpointOutNoMetadata? I don't see it really used anywhere. EndpointOut should just not have metadata and that's it.

Reply:
Why shouldn't EndpointOut return the metadata field. We always return this field in our normal API - beyond that it seems like unnecessary complexity to hide this field normally.

See the PR description for `EndpointNoMetadata` got added - one of the message attempt endpoints uses `EndpointOut`, and we _don't_ normally return the metadata there.

Reply:
OK, I get this comment now, though I'd at least call it EndpointOutCommon or something.